### ISSUE TITLE
PropType Errors on page load

### ISSUE THREAD

**Issue by patricksimonian** [bcgov/devhub-app-web#205]
**Describe the bug**
getting prop type errors

**To Reproduce**
1. Go to home page
2. open dev tools
3. See error

**Expected behavior**
no prop type errors

**Screenshots**
```error
index.js:2177 Warning: Failed prop type: Invalid prop `actions[0].icon` supplied to `ActionsRibbon`, expected a ReactNode.
    in ActionsRibbon (created by Component Card Component)
    in Component Card Component (created by Github Cards Component)
    in div (created by Toggle)
    in div (created by Toggle)
    in Toggle (created by Github Cards Component)
    in div (created by Github Cards Component)
    in section (created by Github Cards Component)
    in Github Cards Component (at pages/index.js:46)
    in Flag (at pages/index.js:106)
    in div (at pages/index.js:105)
    in div (at pages/index.js:102)
    in main (at pages/index.js:60)
    in div (created by Layout)
    in Layout (created by Connect(Layout))
    in Connect(Layout) (at pages/index.js:55)
    in Index (created by Connect(Index))
    in Connect(Index) (created by PageRenderer)
    in PageRenderer (created by JSONStore)
    in JSONStore (created by EnsureResources)
    in ScrollContext (created by EnsureResources)
    in RouteUpdates (created by EnsureResources)
    in EnsureResources (created by RouteHandler)
    in RouteHandler (created by Root)
    in div (created by FocusHandlerImpl)
    in FocusHandlerImpl (created by Context.Consumer)
    in FocusHandler (created by RouterImpl)
    in RouterImpl (created by LocationProvider)
    in LocationProvider (created by Context.Consumer)
    in Location (created by Context.Consumer)
    in Router (created by Root)
    in Root
    in FlagsProvider (created by ConnectedFlagsProvider)
    in ConnectedFlagsProvider
    in Provider
    in _default (created by HotExported_default)
    in AppContainer (created by HotExported_default)
    in HotExported_default
```

**Comment by patricksimonian** [comment]
fixed by #215

**Comment by patricksimonian** [comment]
this issue does not exist anymore, that component was removed.
